.github/workflows/cypress-workflow.yml

```diff
@@ -2,12 +2,13 @@ name: E2E Cypress tests
 on:
   pull_request:
     branches:
-      - "*"
+      - "**"
```

Comment:
For my knowledge, what does the second * do?

Reply:
It ensures that branches that have a `/` will be included for the ci tests